pkg/backplaneapi/clientUtils.go

```diff
@@ -32,16 +32,19 @@ var (
 	DefaultClientUtils ClientUtils = &DefaultClientUtilsImpl{}
 )
 
-func (s *DefaultClientUtilsImpl) MakeRawBackplaneAPIClientWithAccessToken(base, accessToken string) (BackplaneApi.ClientInterface, error) {
-	co := func(client *BackplaneApi.Client) error {
+func makeClientOptions(accessToken string) BackplaneApi.ClientOption {
+	return func(client *BackplaneApi.Client) error {
 		client.RequestEditors = append(client.RequestEditors, func(ctx context.Context, req *http.Request) error {
 			req.Header.Add("Authorization", "Bearer "+accessToken)
 			req.Header.Set("User-Agent", "backplane-cli"+info.Version)
+			req.Header.Set("Backplane-Version", info.DefaultInfoService.GetVersion())
```

Comment:
The version should be included in the above line with `info.Version`?

Reply:
So this also supports `(devel)` and `unknown` in the relevant places rather than being blank. When its blank it will get an older version as we are forced to assume that it is an older client.